robottests/README.rst

```diff
@@ -0,0 +1,28 @@
+.. Copyright (C) 2019, Nokia
+
+
+Running and developing Robot framework tests
+============================================
+
+To run tests, supplement targethosts.py file, which has
+two HOSTS.
+HOSTS are dict objects which need to have the following attributes:
+- host (e.g. localhost)
+- user (e.g. examplename)
+In addition, HOST1 needs to have a regular password (e.g. 'password': 'password')
+while HOST2's needs to have keyfile for ssh authentication.
+e.g. 'key': '/root/.ssh/rsa_keys'
+The keys need to be Ed25519 keys.
+
+In addition, there needs to be a SUDOSHELL dict variable in the targethosts.py
+file which has to following attributes:
+- shellname l)
+To run tests in docker simply type::
+
+   tox -e docker-robottests
+
+and robottests for python 2 and 3 are run inside docker.
+
+To manually run robottests, e.g. for python 2, run the following command::
+
+   tox -e {environment} {posargs}
```

Comment:
Did you mean 

tox -e robottests2 -- {posargs}

It is not wrong to give full example here